projects/rocprofiler-compute/CMakeLists.txt

```diff
@@ -49,6 +49,72 @@ set(EXECUTABLE_NAME "rocprof-compute")
 include(ExternalProject)
 include(GNUInstallDirs)
 
+set(ENABLE_KOKKOS OFF)
+option(USE_MOCK_KOKKOS "Use mock Kokkos implementation for testing" ON)
+
+if(ENABLE_TESTS)
+    if(UNIX)
+        # Detect AMD GPU architecture using rocminfo
+        execute_process(
+            COMMAND
+                bash -c
+                "rocminfo | grep 'Name:' | grep -E 'gfx[0-9]+' | head -n 1 | awk '{print $2}'"
```

Comment:
Assume default installation of rocminfo is not a good idea. Technically, we should retrieve the ROCm root path in cmake.